Makefile

```diff
@@ -24,10 +27,26 @@ build:
 
 ## setup: Runs mod download and generate
 setup:
-	@echo "Downloading tools and dependencies..."
+ifdef DEBIANOS
+	@echo "Downloading packages for Debian based..."
+	@sudo apt-get --allow-releaseinfo-change update
+	@sudo DEBIAN_FRONTEND=noninteractive apt-get install -yq libxcursor-dev libxrandr-dev libxinerama-dev libxi-dev libgl1-mesa-dev libsdl2-dev libasound2-dev xvfb libgtk-3-dev libasound2-dev libxxf86vm-dev
+endif
+ifdef REDHATOS
+	@echo "Downloading packages for RedHat based..."
+	@sudo dnf install -y xorg-x11-server-Xvfb libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel alsa-lib-devel libXi-devel
+endif
+	@echo "Run: git submodule update --init --recursive"
 	@git submodule update --init --recursive
+	@echo "Run: go get -d"
 	@$(GOCMD) get -d
+	@echo "Run: go get -u"
+	@$(GOCMD) get -u
```

Comment:
I don't think, that workflow (and anyone, who uses `make setup`) should update dependencies. It chould cause crasches.

Reply:
True, this is too intrusive. I will separate this in different `make` step.